.goreleaser.yml

```diff
@@ -0,0 +1,62 @@
+---
+before:
+  hooks:
+    - go mod vendor
+project_name: furyagent
+builds:
+  - binary: 'furyagent-darwin-amd64'
+    id: furyagent-darwin
+    goarch:
+      - amd64
+    goos:
+      - darwin
+    env:
+      - CGO_ENABLED=0
+  - binary: 'furyagent-linux-amd64'
+    id: furyagent-linux
+    goarch:
+      - amd64
+    goos:
+      - linux
+    env:
+      - CGO_ENABLED=0
+archives:
+  - format: binary
```

Comment:
@lzecca78 is it possible to have the binary updated as is instead of having it wrapped in a `.tar.gz`?

I am asking this because it's handy to just have the binary somewhere and grab it through an ansible `get_url` task.

Reply:
as you can see the archive format is set to `binary` properly for that purpose :)

Reply:
Fantastic! I should've checked the docs more carefully. 😄